src/CLU/Microsoft.ScenarioTests.CLU/out.txt

```diff
@@ -0,0 +1,33 @@
+SETLOCAL EnableExtensions
+
+ECHO Test return code success case
+call testclu success show 2> out.error
+REM We expect that the commands above succeed (return 0)
+IF %ERRORLEVEL% NEQ 0 (
+	ECHO "Expected error level 0 for success, got " %ERRORLEVEL%
+	EXIT /B 1
+)
+
+ECHO Test return codes non terminating error case
+call testclu success show --generatenonterminatingerror true 2> out.error
+IF %ERRORLEVEL% NEQ 1 (
+	ECHO "Expected error level 1 for non-terminating error, got " %ERRORLEVEL%
+	EXIT /B 1
+)
+
+ECHO Test return codes terminating error case
+call testclu success show --generateterminatingerror true 2> out.error
+IF %ERRORLEVEL% NEQ 2 (
+	ECHO "Expected error level 2 for terminating error, got " %ERRORLEVEL%
```

Comment:
Ick. That file shouldn’t have been included. Removed.

From: Yugang Wang [mailto:notifications@github.com]
Sent: Thursday, December 17, 2015 4:42 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Johan Stenberg <Johan.Stenberg@microsoft.com>
Subject: Re: [azure-powershell] Update tests + escape paramters passed to clurun by (bash) wrapper script (#1519)


In src/CLU/Microsoft.ScenarioTests.CLU/out.txt<https://github.com/Azure/azure-powershell/pull/1519#discussion_r47982181>:

> +IF %ERRORLEVEL% NEQ 0 (

> +     ECHO "Expected error level 0 for success, got " %ERRORLEVEL%

> +     EXIT /B 1

> +)

> +

> +ECHO Test return codes non terminating error case

> +call testclu success show --generatenonterminatingerror true 2> out.error

> +IF %ERRORLEVEL% NEQ 1 (

> +     ECHO "Expected error level 1 for non-terminating error, got " %ERRORLEVEL%

> +     EXIT /B 1

> +)

> +

> +ECHO Test return codes terminating error case

> +call testclu success show --generateterminatingerror true 2> out.error

> +IF %ERRORLEVEL% NEQ 2 (

> +     ECHO "Expected error level 2 for terminating error, got " %ERRORLEVEL%

the tab is also not right in this file

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1519/files#r47982181>.